pkg/controllers/cluster/taint_manager.go

```diff
@@ -245,11 +253,14 @@ func (tc *NoExecuteTaintManager) syncClusterBindingEviction(key util.QueueKey) e
 // needEviction returns whether the binding should be evicted from target cluster right now.
 // If a toleration time is found, we return false along with a minimum toleration time as the
 // second return value.
-func (tc *NoExecuteTaintManager) needEviction(clusterName string, appliedPlacement map[string]string) (bool, time.Duration, error) {
-	placement, err := helper.GetAppliedPlacement(appliedPlacement)
+func (tc *NoExecuteTaintManager) needEviction(clusterName string, annotations map[string]string) (bool, time.Duration, error) {
+	placement, err := helper.GetAppliedPlacement(annotations)
 	if err != nil {
 		return false, -1, err
 	}
+	if placement == nil {
+		return false, -1, fmt.Errorf("the applied placement for ResourceBining can not be empty")
+	}
```

Comment:
Can you remind me in what situations the placement could be nil?

Reply:
This problem does not occur in the current implementation of Karmada, but it depends on the scheduler. If the user does not use the default scheduler, this problem may occur.

Reply:
Got, then please add a comment on it, otherwise, it brings misunderstanding.

Reply:
Updated.